src/Illuminate/Queue/Console/RetryCommand.php

```diff
@@ -31,7 +32,7 @@ public function fire()
         $ids = $this->argument('id');
 
         if (count($ids) === 1 && $ids[0] === 'all') {
-            $ids = collect($this->laravel['queue.failer']->all())->pluck('id')->all();
+            $ids = (new Collection($this->laravel['queue.failer']->all()))->pluck('id')->all();
```

Comment:
fwiw, I would simply do `array_column($this->laravel['queue.failer']->all(), 'id')`